src/components/AttachmentCarousel/index.js

```diff
@@ -252,7 +252,8 @@ class AttachmentCarousel extends React.Component {
      * @returns {JSX.Element}
      */
     renderCell(props) {
-        const style = [props.style, styles.h100, {width: this.state.containerWidth}];
+        // We are isolating the method this function uses to update the width of the container to make it respond faster
+        const style = [props.style, styles.h100, {width: useWindowDimensions().width}];
```

Comment:
The idea here is to make some space between the container and the edges of the screen: 
https://github.com/Expensify/App/blob/2c1dfe94ce5d69a44af4a8d384903bc85a7c2e00/src/styles/getModalStyles/getBaseModalStyles.js#L114-L117
Notice that for mobile we are using a width on the border and the idea is to make some space for mobile devices, however looking at this: 
https://github.com/Expensify/App/blob/2c1dfe94ce5d69a44af4a8d384903bc85a7c2e00/src/components/AttachmentCarousel/index.js#L288-L289
We are adding 1 under the assumption that we have a border that we need to compensate for which we don't since the border width is only added for small screens so I think that + 1 shouldn't be there because it's adding an extra 1px to web, the reason it's 1px smaller in small screens is because border width will be applied to both sides removing 2px and then we add 1px in the above so mobile becomes 1px smaller while web 1px bigger.